src/de/hhn/aib/labsw/blackmirror/controller/API/websockets/SendPackage.java

```diff
@@ -1,8 +1,11 @@
-package de.hhn.aib.labsw.blackmirror.controller.API;
+package de.hhn.aib.labsw.blackmirror.controller.API.websockets;
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 
+/**
+ * Dataclass to hold the Data of a Package send by the API
+ */
 public class SendPackage {
```

Comment:
instead of using a class (or "dataclass") use a Java record, see "https://docs.oracle.com/en/java/javase/17/language/records.html"

'public class SendPackage { ... }' becomes 'record SendPackage(String topic, JsonNode payload)'